ValveResourceFormat/Resource/ResourceTypes/Model.cs

```diff
@@ -172,5 +172,15 @@ public IEnumerable<bool> GetActiveMeshMaskForGroup(string groupName)
                 return meshGroupMasks.Select(_ => false);
             }
         }
+
+        public IKeyValueCollection GetMorphData()
+        {
+            if (Resource.GetBlockByType(BlockType.MRPH) is KeyValuesOrNTRO mrph)
```

Comment:
This will be wrong for models with multiple meshes/morphs.

You could update GetEmbeddedMeshes() to add `morph_block` to the `Mesh` class.

Reply:
Yeah, I am working this.